docs/pages/admin-guides/teleport-policy/integrations/entra-id.mdx

```diff
@@ -90,7 +141,276 @@ Back in the Teleport Web UI, fill out the required data and click "Finish".
 ![Second step of the Entra ID integration onboarding with required fields filled in](../../../../img/access-graph/entra-id/integration-wizard-step-2-filled.png)
 </Figure>
 
-## Step 3/3. Analyze Entra ID directory in Teleport Access Graph
+</TabItem>
+
+<TabItem label="Automatic setup with system credentials" >
+
+### Step 1/2. Assign permissions to Azure Identity of your Auth Service VMs
+
+To set up the Azure Identity with the necessary permissions:
+
+- `Application.Read.All`
+- `Directory.Read.All`
+- `Policy.Read.All`
+
+Go to your Azure Dashboard, find the identities linked to your Teleport Auth Service VMs,
+and copy the `Object (principal) ID`. Paste this value into `<Var name="Principal ID" />`.
+
+After obtaining the Principal ID, open the [Azure Cloud Shell](https://portal.azure.com/#cloudshell/)
+in PowerShell mode and run the following script to assign the required permissions to `<Var name="Principal ID" />`.
+
+```text
```

Comment:
>Nit: Can the script be placed under collapsible section?

I added them but preferred them to be opened by default. I found it quite easy to skip the scripts because our collapsible sections are not very visible  when collapsed

collapsed:
<img width="866" alt="Screenshot 2024-11-08 at 11 03 04" src="https://github.com/user-attachments/assets/0430ef60-a53b-4854-bae0-ed681c7cfc14">

expanded:

<img width="903" alt="Screenshot 2024-11-08 at 11 03 43" src="https://github.com/user-attachments/assets/cbcd75a9-cc73-4052-b910-b7281bc139ea">

> Also, is there a powershell code fence type for syntax highlighting?

yes, I thought we still didn't support `<Var/>` in code blocks